tests/test_resources/test_with_default_tag/packager-config.yml

```diff
@@ -0,0 +1,80 @@
+bundle:
```

Comment:
IIRC the plan was to support multiple tests (Jenkinsfiles) for a single configuration. The current structure does not seem to reflect it.

Reply:
Actually the paths for both the configuration and the Jenkinsfile are provided when calling the test, so why do you think it's not possible?

Reply:
@oleg-nenashev it's just a test yaml as example. The real tests are not already implemented